docs/cql/tombstone-gc.rst

```diff
@@ -0,0 +1,109 @@
+=====================================
+Tombstone Garbage Collection (GC) 
+=====================================
+
+Overview
+----------
+When you delete some data from the database, the data record is not immediately 
+removed. Instead, it is marked with a tombstone - a data marker that prevents 
+the data from being returned as a query result. The data covered by the tombstone 
+and the tombstone itself are eventually removed after the compaction process 
+is performed.
+
+ScyllaDB allows you to manage how and when tombstones are removed, considering 
+that tombstones must be garbage-collected:
+
+* Within a reasonable time because a large number of tombstones impacts query 
+  response times and leads to performance degradation.
+* After a cluster-wide repair to ensure that all replicas are in sync. Otherwise, 
+  tombstones might be missing on some replica nodes, which may lead to 
+  the resurrection of deleted data.
+
+You can choose one of the following ways to garbage-collect tombstones:
+
+* Repair-based Tombstone GC (recommended) - tombstones are removed when a repair 
```

Comment:
maybe remind the reader that repair-based tombstone GC is recommended, but not currently the default and need to be manually enabled on each table. (@asias is there a global option to enable it for all new tables?)

Reply:
No. New table is still not default to use  tomgstone_gc = repair yet and there is no such options.